openhantek/CMakeLists.txt

```diff
@@ -29,8 +29,13 @@ add_subdirectory(translations)
 
 add_definitions(-DVERSION="${CPACK_PACKAGE_VERSION}")
 
+set(EXECTYPE "")
+if(WIN32 AND MINGW)
```

Comment:
Shouldn't it be WIN32 OR MINGW? In both cases we need the "WIN32" argument for `add_executable`, or am I mistaken here?

Reply:
Mmm I can make mistake too
but I think in the next way
WIN32 is target 
MINGW what we are using to build

Under MINGW we can built for linux so in case of OR we will end with passing win32 as parameter for windows and for linux I can't predict the result :)

As visual studio compiled it fine  without this parameter (suppose it was compiled :) really i don't know) if we put OR again we will the pass this parameter for VS.

And again Mac it's possible to use mingw there in case of OR again parameter will be passed, again unpredictable behavior

So that's why I've put AND do not break what there is already in script 

But again maybe I'm wrong )

Reply:
"MinGW provides a complete Open Source programming tool set which is suitable for the development of native MS-Windows applications"

MinGW is for building windows applications and those need the WIN32 parameter for `add_executable`, no matter on which operating system you are cross compiling them. That's how I understand this matter.

Reply:
Mmm yes suppose you are right - in any case target will be win32 ) 
so then it's  just enough to remove win32 and then visual studio still will be without this parameter